build/scss/colors.scss

```diff
@@ -50,6 +50,6 @@ $grey500: #596D84;
 $grey600: #475A70;
 $grey700: #36485C;
 $grey800: #2E3A47;
-$white: #FFFFFF;
-$black: #000000;
-$transparent: transparent;
\ No newline at end of file
+$shadesWhite: #FFFFFF;
+$shadesBlack: #000000;
+$shadesTransparent: transparent;
```

Comment:
this is back to yucky, but leaving for now since we're going to have to think about owerly on it's own anyways